docs/src/use/configure/migration-guide.md

```diff
@@ -0,0 +1,339 @@
+    parserOptions: {
+        ecmaVersion: 2022,
+        sourceType: "module"
+    }
+    // ...other config
+}
+```
+
+#### eslint.config.js
+
+In the `eslint.config.js`, the `globals`, and `parserOptions` are consolidated under the `languageOptions` key.
+
+There is no longer the `env` property in `eslint.config.js`. Global variables for specific runtimes are imported from the [globals](https://www.npmjs.com/package/globals) npm package and included in the `globals` property. You can use the spread operator (`...`)to import all globals:
+
+```javascript
+// eslint.config.js
+import globals from "globals";
+
+export default [
+    {
+        languageOptions: {
+            globals: {
+                ...globals.browser,
+                myCustomGlobal: "readonly"
+            },
+            parserOptions: {
+                ecmaVersion: 2022,
+                sourceType: "module"
+            }
+        }
+        // ...other config
+    }
+];
+```
+
+### Predefined Configs
+
+#### .eslintrc
+
+To use predefined configs, use the `extends` property:
+
+```javascript
+// .eslintrc.js
+module.exports = {
+    // ...other config
+    extends: "eslint:recommended",
+    rules: {
+        semi: ["warn", "always"]
+    },
+    // ...other config
+}
+```
+
+ESLint comes with two predefined configs:
+
+* -`eslint:recommended`: the rules recommended by ESLint
+* -`eslint:all`: all rules shipped with ESLint
+
+You can also use the `extends` property to extend a custom config. Custom configs can either be paths to local config files or npm package names:
+
+```javascript
+// .eslintrc.js
+module.exports = {
+    // ...other config
+    extends: ["eslint:recommended", "./custom-config.js", "eslint-config-my-config"],
+    rules: {
+        semi: ["warn", "always"]
+    },
+    // ...other config
+}
+```
+
+#### eslint.config.js
+
+`eslint.config.js` supports the built-in `eslint:recommended` and `eslint:all` configs, locally defined configs and npm package configs.
+
+You can add each of these configs to the exported array.
+
+You must import the modules for local config files and npm package configs with `eslint.config.js`:
+
+```javascript
+// eslint.config.js
+
+import customConfig from "./custom-config.js";
+import myConfig from "eslint-config-my-config";
+
+export default [
+    "eslint:recommended",
+    customConfig,
+    myConfig,
+    {
+        rules: {
+            semi: ["warn", "always"]
+        },
+        // ...other config
+    }
+];
+```
+
+Note that because you are just importing JavaScript modules, you can mutate the config objects before exporting them. For example, you might want to have a certain config object only apply to your test files:
+
+```javascript
+// eslint.config.js
+import customTestConfig from "./custom-test-config.js";
+export default [
+    "eslint:recommended",
+    {
+        ...customTestConfig,
+        files: ["**/*.test.js"],
+    },
+];
+```
+
+## Things That Haven’t Changed between Configuration File Formats
```

Comment:
to reviewer: other notable things which should be highlighted?

Reply:
I think we should put this section first, to let people know "these are the things you don't have to worry about".

I think it's also helpful to point out that `globals` are configured the same way, just in a different location.